Makefile.local.mk

```diff
@@ -0,0 +1,28 @@
+##@ Chart
+
+MAIN_CHART=helm/linkerd-viz
+VENDOR_CHART_LINKERD-VIZ=vendor/linkerd/viz/charts/linkerd-viz
+VENDOR_CHART_PARTIALS=vendor/linkerd/charts/partials
+
+
+.PHONY: update-chart helm-docs apply-vendor update-deps
+update-chart: ## Run vendir sync
+	@echo "====> $@"
+	vendir sync
+	$(MAKE) apply-vendor
+# 	$(MAKE) update-deps
+
+#update-deps:
+#	new_version=`yq .version $(MAIN_CHART)/charts/linkerd-crds/Chart.yaml` && \
```

Comment:
since this chart has no deps other than `partials`, you can remove this line and the following and reenable the task (so it runs `helm dependency update`)